docs/content/documentation/systems/Distributions/index.md

```diff
@@ -0,0 +1,15 @@
+# Distributions System
+
+The distribution system is a new statistical libraries in MOOSE.
+It provides support for several probability distributions. Currently,
```

Comment:
Please remove the "Currently, the use can only..." This will likely change soon.